csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5102,6 +5102,56 @@ The relevant paths for this test are:
 
 > The version range `>4.2` is a valid vsl but not valid according to the vers specification.
 
+### 6.2.19 CVSS for Fixed Products
+
+For each item the fixed products group (`first_fixed` and `fixed`) it MUST be tested that a CVSS applying to this product has an environmental score of `0`. The test SHALL pass if none of the Product IDs in fixed products group is listed in `products` of any item of the scores element.
```

Comment:
I am ok, but we are juggling some things and the "missing" the does not help. What about instead of:
```
The test SHALL pass if none of the Product IDs in fixed products group is listed in `products` of any item of the scores element.
```
write:
```
The test SHALL pass if none of the Product IDs listed within product status `fixed` or `first_fixed` is found in `products` of any item of the scores element.
```

Reply:
Done.